hack/lib/mesh.bash

```diff
@@ -89,6 +89,9 @@ function deploy_gateways {
       --key="${out_dir}"/wildcard.key \
       --cert="${out_dir}"/wildcard.crt --dry-run=client -o yaml | oc apply -f -
 
+  if ! oc get namespace "${SERVING_NAMESPACE}" &>/dev/null; then
```

Comment:
Could you do such fixes in a separate PR to be confident. I know tests pass but this PR is quite big.

Reply:
OK. I can probably pull this out.

Reply:
Alright, I sent two other PRs separately:
https://github.com/openshift-knative/serverless-operator/pull/1510
https://github.com/openshift-knative/serverless-operator/pull/1511
And squashed some commits and force-pushed.